2014-06-19 14:15:38,683 - [INFO] - from play in pool-4-thread-1 
Listening for HTTP on /0:0:0:0:0:0:0:0:9000

2014-06-19 14:16:32,886 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:16:33,257 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:16:45,693 - [ERROR] - from application in New I/O worker #1 


! @6ii8gm2ge - Internal server error, for (GET) [/] ->

play.api.db.evolutions.InvalidDatabaseRevision: Database 'default' needs evolution![An SQL script need to be run on your database.]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1$$anonfun$apply$1.apply$mcV$sp(Evolutions.scala:486) ~[play-jdbc_2.10.jar:2.2.3]
	at play.api.db.evolutions.EvolutionsPlugin.withLock(Evolutions.scala:507) ~[play-jdbc_2.10.jar:2.2.3]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1.apply(Evolutions.scala:461) ~[play-jdbc_2.10.jar:2.2.3]
	at play.api.db.evolutions.EvolutionsPlugin$$anonfun$onStart$1.apply(Evolutions.scala:459) ~[play-jdbc_2.10.jar:2.2.3]
	at scala.collection.immutable.List.foreach(List.scala:318) ~[scala-library.jar:na]
	at play.api.db.evolutions.EvolutionsPlugin.onStart(Evolutions.scala:459) ~[play-jdbc_2.10.jar:2.2.3]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.3]
	at play.api.Play$$anonfun$start$1$$anonfun$apply$mcV$sp$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.3]
	at scala.collection.immutable.List.foreach(List.scala:318) ~[scala-library.jar:na]
	at play.api.Play$$anonfun$start$1.apply$mcV$sp(Play.scala:88) ~[play_2.10.jar:2.2.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.3]
	at play.api.Play$$anonfun$start$1.apply(Play.scala:88) ~[play_2.10.jar:2.2.3]
	at play.utils.Threads$.withContextClassLoader(Threads.scala:18) ~[play_2.10.jar:2.2.3]
	at play.api.Play$.start(Play.scala:87) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:139) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1$$anonfun$1.apply(ApplicationProvider.scala:112) ~[play_2.10.jar:2.2.3]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:112) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1$$anonfun$apply$1.apply(ApplicationProvider.scala:110) ~[play_2.10.jar:2.2.3]
	at scala.util.Success.flatMap(Try.scala:200) ~[scala-library.jar:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:110) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 14:16:58,409 - [INFO] - from play in play-internal-execution-context-1 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:16:58,413 - [INFO] - from play in play-internal-execution-context-1 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:16:59,166 - [INFO] - from play in play-internal-execution-context-1 
Application started (Dev)

2014-06-19 14:21:18,070 - [INFO] - from play in play-internal-execution-context-2 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:21:18,078 - [INFO] - from play in play-internal-execution-context-2 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:21:18,504 - [INFO] - from play in play-internal-execution-context-2 
Application started (Dev)

2014-06-19 14:21:26,260 - [INFO] - from play in play-internal-execution-context-2 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:21:26,265 - [INFO] - from play in play-internal-execution-context-2 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:21:26,605 - [INFO] - from play in play-internal-execution-context-2 
Application started (Dev)

2014-06-19 14:22:01,489 - [INFO] - from play in play-internal-execution-context-3 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:22:01,493 - [INFO] - from play in play-internal-execution-context-3 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:22:01,828 - [INFO] - from play in play-internal-execution-context-3 
Application started (Dev)

2014-06-19 14:25:40,232 - [INFO] - from play in play-internal-execution-context-3 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:25:40,237 - [INFO] - from play in play-internal-execution-context-3 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:25:40,632 - [INFO] - from play in play-internal-execution-context-3 
Application started (Dev)

2014-06-19 14:26:11,320 - [INFO] - from play in play-internal-execution-context-3 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:26:11,326 - [INFO] - from play in play-internal-execution-context-3 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:26:11,673 - [INFO] - from play in play-internal-execution-context-3 
Application started (Dev)

2014-06-19 14:27:13,310 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:27:13,315 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:27:13,615 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:28:50,693 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:28:50,698 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:28:51,001 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:29:22,596 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:29:22,601 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:29:22,895 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:29:58,177 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:29:58,180 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:29:58,438 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:40:29,286 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:40:29,290 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:40:29,559 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:44:30,874 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:44:30,878 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:44:31,093 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:46:05,432 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:46:05,436 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:46:05,765 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:49:49,218 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:49:49,223 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:49:49,580 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:51:25,724 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:51:25,728 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:51:25,944 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:51:42,406 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:51:42,413 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:51:42,706 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:53:49,757 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:53:49,766 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:53:50,040 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:54:28,978 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:54:28,982 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:54:29,193 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:55:47,018 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:55:47,022 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:55:47,260 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:56:15,522 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:56:15,527 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:56:15,747 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:56:34,963 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:56:34,970 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:56:35,204 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:57:01,054 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:57:01,061 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:57:01,302 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 14:58:22,261 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 14:58:22,266 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 14:58:22,507 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:01:35,389 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:01:35,392 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:01:35,618 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:02:05,990 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:02:06,034 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:02:06,276 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:03:56,882 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:03:56,885 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:03:57,077 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:08:22,167 - [ERROR] - from application in New I/O worker #2 


! @6ii8ng6m5 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$TemplateCompilationException: Compilation error[Unmatched bracket]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:55) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:30) ~[na:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:35) ~[scala-library.jar:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.8.0_05]
	at java.lang.Thread.run(Unknown Source) [na:1.8.0_05]

2014-06-19 15:08:43,145 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:08:43,149 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:08:43,477 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:11:19,598 - [ERROR] - from application in New I/O worker #6 


! @6ii8o09ij - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value listHack]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:11:40,348 - [ERROR] - from application in New I/O worker #6 


! @6ii8o1eam - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[illegal start of simple expression]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:12:15,274 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:12:15,277 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:12:15,544 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:35:56,803 - [ERROR] - from application in New I/O worker #7 


! @6ii916ao6 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[illegal start of simple expression]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:36:47,400 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:36:47,404 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:36:47,646 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:37:46,531 - [INFO] - from play in play-internal-execution-context-6 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:37:46,534 - [INFO] - from play in play-internal-execution-context-6 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:37:46,740 - [INFO] - from play in play-internal-execution-context-6 
Application started (Dev)

2014-06-19 15:38:28,446 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 15:38:28,450 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 15:38:28,756 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 15:45:43,547 - [ERROR] - from application in New I/O worker #6 


! @6ii92dkn8 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:46:35,289 - [ERROR] - from application in New I/O worker #6 


! @6ii92gjba - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:47:43,774 - [ERROR] - from application in New I/O worker #6 


! @6ii92kgjc - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:49:58,688 - [ERROR] - from application in New I/O worker #6 


! @6ii93288d - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:50:24,241 - [ERROR] - from application in New I/O worker #6 


! @6ii933k37 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value form]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:50:59,716 - [ERROR] - from application in New I/O worker #6 


! @6ii935kfi - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[value newEvent is not a member of controllers.ReverseApplication]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:53:25,835 - [ERROR] - from application in New I/O worker #6 


! @6ii93e2jh - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[value newEvent is not a member of controllers.ReverseApplication]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 15:57:56,039 - [ERROR] - from application in New I/O worker #6 


! @6ii943cc3 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[value newEvent is not a member of controllers.ReverseApplication]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 16:00:02,586 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:00:02,589 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:00:02,851 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:01:49,575 - [ERROR] - from application in New I/O worker #6 


! @6ii94gjo7 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value textBox]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 16:01:59,763 - [ERROR] - from application in New I/O worker #6 


! @6ii94h903 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value TextBox]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 16:02:13,594 - [ERROR] - from application in New I/O worker #6 


! @6ii94i3c3 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$CompilationException: Compilation error[not found: value boxText]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14$$anonfun$apply$16.apply(PlayReloader.scala:304) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:304) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2$$anonfun$apply$14.apply(PlayReloader.scala:298) ~[na:na]
	at scala.Option.map(Option.scala:145) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:298) ~[na:na]
	at play.PlayReloader$$anon$1$$anonfun$reload$2.apply(PlayReloader.scala:296) ~[na:na]
	at scala.util.Either$LeftProjection.map(Either.scala:377) ~[scala-library.jar:na]
	at play.PlayReloader$$anon$1.reload(PlayReloader.scala:296) ~[na:na]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:104) ~[play_2.10.jar:2.2.3]
	at play.core.ReloadableApplication$$anonfun$get$1.apply(ApplicationProvider.scala:102) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.liftedTree1$1(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.impl.Future$PromiseCompletingRunnable.run(Future.scala:24) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) ~[scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) ~[scala-library.jar:na]

2014-06-19 16:04:51,724 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:04:51,727 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:04:51,972 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:05:49,758 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:05:49,763 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:05:50,039 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:07:06,942 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:07:06,946 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:07:07,224 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:20:24,440 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:20:24,454 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:20:24,723 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:20:51,740 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:20:51,744 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:20:51,960 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:21:10,213 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:21:10,216 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:21:10,472 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:23:53,511 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:23:53,515 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:23:53,808 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:24:22,191 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:24:22,197 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:24:22,436 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:38:40,036 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:38:40,039 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:38:40,466 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:41:04,331 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:41:04,334 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:41:04,589 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:41:22,398 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:41:22,402 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:41:22,676 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:52:47,555 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:52:47,563 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:52:47,970 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:57:49,690 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:57:49,694 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:57:50,604 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:58:23,645 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:58:23,647 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:58:23,906 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 16:58:55,231 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 16:58:55,234 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 16:58:55,590 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 17:18:01,308 - [ERROR] - from application in New I/O worker #5 


! @6ii9egml0 - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$TemplateCompilationException: Compilation error[Unmatched bracket]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:55) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:30) ~[na:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:35) ~[scala-library.jar:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.8.0_05]
	at java.lang.Thread.run(Unknown Source) [na:1.8.0_05]

2014-06-19 17:20:03,500 - [ERROR] - from application in New I/O worker #5 


! @6ii9enlel - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$TemplateCompilationException: Compilation error[Unmatched bracket]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:55) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:30) ~[na:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:35) ~[scala-library.jar:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.8.0_05]
	at java.lang.Thread.run(Unknown Source) [na:1.8.0_05]

2014-06-19 17:20:06,810 - [ERROR] - from application in New I/O worker #5 


! @6ii9eo0bl - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$TemplateCompilationException: Compilation error[Unmatched bracket]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:55) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:30) ~[na:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:35) ~[scala-library.jar:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.8.0_05]
	at java.lang.Thread.run(Unknown Source) [na:1.8.0_05]

2014-06-19 17:21:43,063 - [ERROR] - from application in New I/O worker #5 


! @6ii9f3cma - Internal server error, for (GET) [/hackfest] ->

play.PlayExceptions$TemplateCompilationException: Compilation error[Unmatched bracket]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:55) ~[na:na]
	at play.PlaySourceGenerators$$anonfun$3.apply(PlaySourceGenerators.scala:30) ~[na:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:35) ~[scala-library.jar:na]
	at scala.Function5$$anonfun$tupled$1.apply(Function5.scala:34) ~[scala-library.jar:na]
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47) ~[scala-library.jar:na]
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:42) ~[na:na]
	at sbt.std.Transform$$anon$4.work(System.scala:64) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:18) ~[na:na]
	at sbt.Execute.work(Execute.scala:244) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:237) ~[na:na]
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:160) ~[na:na]
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:30) ~[na:na]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.Executors$RunnableAdapter.call(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.FutureTask.run(Unknown Source) ~[na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) [na:1.8.0_05]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) [na:1.8.0_05]
	at java.lang.Thread.run(Unknown Source) [na:1.8.0_05]

2014-06-19 17:22:30,281 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 17:22:30,284 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 17:22:30,520 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 17:22:30,794 - [ERROR] - from play in play-internal-execution-context-23 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: Cannot instantiate class models.User. It must have a default constructor

2014-06-19 17:22:31,247 - [ERROR] - from application in play-internal-execution-context-23 


! @6ii9f6620 - Internal server error, for (GET) [/hackfest] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: Cannot instantiate class models.User. It must have a default constructor]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.3]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:260) [play_2.10.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.3]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: Cannot instantiate class models.User. It must have a default constructor
	at play.data.Form.blankInstance(Form.java:90) ~[play-java_2.10.jar:2.2.3]
	at play.data.Form.field(Form.java:649) ~[play-java_2.10.jar:2.2.3]
	at play.data.Form.apply(Form.java:609) ~[play-java_2.10.jar:2.2.3]
	at views.html.index$$anonfun$apply$2$$anonfun$apply$4.apply(index.template.scala:121) ~[na:na]
	at views.html.index$$anonfun$apply$2$$anonfun$apply$4.apply(index.template.scala:120) ~[na:na]
	at views.html.helper.form$.apply(form.template.scala:45) ~[play_2.10.jar:2.2.3]
	at views.html.index$$anonfun$apply$2.apply(index.template.scala:120) ~[na:na]
	at views.html.index$$anonfun$apply$2.apply(index.template.scala:99) ~[na:na]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244) ~[scala-library.jar:na]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244) ~[scala-library.jar:na]
	at scala.collection.Iterator$class.foreach(Iterator.scala:727) ~[scala-library.jar:na]
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1157) ~[scala-library.jar:na]
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72) ~[scala-library.jar:na]
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54) ~[scala-library.jar:na]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244) ~[scala-library.jar:na]
	at scala.collection.AbstractTraversable.map(Traversable.scala:105) ~[scala-library.jar:na]
	at views.html.index$.apply(index.template.scala:99) ~[na:na]
	at views.html.index$.render(index.template.scala:145) ~[na:na]
	at views.html.index.render(index.template.scala) ~[na:na]
	at controllers.Application.hacks(Application.java:28) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$2$$anonfun$apply$2.apply(routes_routing.scala:69) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$2$$anonfun$apply$2.apply(routes_routing.scala:69) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.3]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction$1.apply(TransactionalAction.java:20) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction$1.apply(TransactionalAction.java:18) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.JPA.withTransactionAsync(JPA.java:177) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction.call(TransactionalAction.java:15) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.3]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted
Caused by: java.lang.InstantiationException: models.User
	at java.lang.Class.newInstance(Unknown Source) ~[na:1.8.0_05]
	at play.data.Form.blankInstance(Form.java:88) ~[play-java_2.10.jar:2.2.3]
	... 42 common frames omitted
Caused by: java.lang.NoSuchMethodException: models.User.<init>()
	at java.lang.Class.getConstructor0(Unknown Source) ~[na:1.8.0_05]
	... 44 common frames omitted

2014-06-19 17:23:13,124 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 17:23:13,128 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 17:23:13,358 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 17:23:13,427 - [ERROR] - from play in play-internal-execution-context-23 
Cannot invoke the action, eventually got an error: java.lang.RuntimeException: Cannot instantiate class models.User. It must have a default constructor

2014-06-19 17:23:13,440 - [ERROR] - from application in play-internal-execution-context-23 


! @6ii9f8gf4 - Internal server error, for (GET) [/hackfest] ->

play.api.Application$$anon$1: Execution exception[[RuntimeException: Cannot instantiate class models.User. It must have a default constructor]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.3]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:260) [play_2.10.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.3]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.RuntimeException: Cannot instantiate class models.User. It must have a default constructor
	at play.data.Form.blankInstance(Form.java:90) ~[play-java_2.10.jar:2.2.3]
	at play.data.Form.field(Form.java:649) ~[play-java_2.10.jar:2.2.3]
	at play.data.Form.apply(Form.java:609) ~[play-java_2.10.jar:2.2.3]
	at views.html.index$$anonfun$apply$2$$anonfun$apply$4.apply(index.template.scala:122) ~[na:na]
	at views.html.index$$anonfun$apply$2$$anonfun$apply$4.apply(index.template.scala:121) ~[na:na]
	at views.html.helper.form$.apply(form.template.scala:45) ~[play_2.10.jar:2.2.3]
	at views.html.index$$anonfun$apply$2.apply(index.template.scala:121) ~[na:na]
	at views.html.index$$anonfun$apply$2.apply(index.template.scala:99) ~[na:na]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244) ~[scala-library.jar:na]
	at scala.collection.TraversableLike$$anonfun$map$1.apply(TraversableLike.scala:244) ~[scala-library.jar:na]
	at scala.collection.Iterator$class.foreach(Iterator.scala:727) ~[scala-library.jar:na]
	at scala.collection.AbstractIterator.foreach(Iterator.scala:1157) ~[scala-library.jar:na]
	at scala.collection.IterableLike$class.foreach(IterableLike.scala:72) ~[scala-library.jar:na]
	at scala.collection.AbstractIterable.foreach(Iterable.scala:54) ~[scala-library.jar:na]
	at scala.collection.TraversableLike$class.map(TraversableLike.scala:244) ~[scala-library.jar:na]
	at scala.collection.AbstractTraversable.map(Traversable.scala:105) ~[scala-library.jar:na]
	at views.html.index$.apply(index.template.scala:99) ~[na:na]
	at views.html.index$.render(index.template.scala:146) ~[na:na]
	at views.html.index.render(index.template.scala) ~[na:na]
	at controllers.Application.hacks(Application.java:28) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$2$$anonfun$apply$2.apply(routes_routing.scala:69) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$2$$anonfun$apply$2.apply(routes_routing.scala:69) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.3]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction$1.apply(TransactionalAction.java:20) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction$1.apply(TransactionalAction.java:18) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.JPA.withTransactionAsync(JPA.java:177) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction.call(TransactionalAction.java:15) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.3]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted
Caused by: java.lang.InstantiationException: models.User
	at java.lang.Class.newInstance(Unknown Source) ~[na:1.8.0_05]
	at play.data.Form.blankInstance(Form.java:88) ~[play-java_2.10.jar:2.2.3]
	... 42 common frames omitted
Caused by: java.lang.NoSuchMethodException: models.User.<init>()
	at java.lang.Class.getConstructor0(Unknown Source) ~[na:1.8.0_05]
	... 44 common frames omitted

2014-06-19 17:24:03,627 - [INFO] - from play in play-internal-execution-context-23 
datasource [jdbc:h2:mem:play] bound to JNDI as DefaultDS

2014-06-19 17:24:03,631 - [INFO] - from play in play-internal-execution-context-23 
database [default] connected at jdbc:h2:mem:play

2014-06-19 17:24:03,926 - [INFO] - from play in play-internal-execution-context-23 
Application started (Dev)

2014-06-19 17:24:28,598 - [ERROR] - from play in play-internal-execution-context-23 
Cannot invoke the action, eventually got an error: java.lang.IllegalStateException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: models.User

2014-06-19 17:24:28,606 - [ERROR] - from application in play-internal-execution-context-23 


! @6ii9fcnk7 - Internal server error, for (POST) [/addUser/1/add] ->

play.api.Application$$anon$1: Execution exception[[IllegalStateException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: models.User]]
	at play.api.Application$class.handleError(Application.scala:293) ~[play_2.10.jar:2.2.3]
	at play.api.DefaultApplication.handleError(Application.scala:399) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3$$anonfun$applyOrElse$3.apply(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at scala.Option.map(Option.scala:145) [scala-library.jar:na]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:264) [play_2.10.jar:2.2.3]
	at play.core.server.netty.PlayDefaultUpstreamHandler$$anonfun$3.applyOrElse(PlayDefaultUpstreamHandler.scala:260) [play_2.10.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:344) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$recoverWith$1.apply(Future.scala:343) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.api.libs.iteratee.Execution$$anon$1.execute(Execution.scala:43) [play-iteratees_2.10.jar:2.2.3]
	at scala.concurrent.impl.CallbackRunnable.executeWithValue(Promise.scala:40) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.tryComplete(Promise.scala:248) [scala-library.jar:na]
	at scala.concurrent.Promise$class.complete(Promise.scala:55) [scala-library.jar:na]
	at scala.concurrent.impl.Promise$DefaultPromise.complete(Promise.scala:153) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$map$1.apply(Future.scala:235) [scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask$AdaptedRunnableAction.exec(ForkJoinTask.java:1361) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1339) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [scala-library.jar:na]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [scala-library.jar:na]
Caused by: java.lang.IllegalStateException: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: models.User
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1384) ~[hibernate-entitymanager-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1310) ~[hibernate-entitymanager-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.ejb.AbstractEntityManagerImpl.convert(AbstractEntityManagerImpl.java:1316) ~[hibernate-entitymanager-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.ejb.AbstractEntityManagerImpl.flush(AbstractEntityManagerImpl.java:999) ~[hibernate-entitymanager-4.2.3.Final.jar:4.2.3.Final]
	at models.DAOEvent.flush(DAOEvent.java:17) ~[na:na]
	at controllers.Application.addUser(Application.java:39) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$4$$anonfun$apply$4.apply(routes_routing.scala:85) ~[na:na]
	at Routes$$anonfun$routes$1$$anonfun$applyOrElse$4$$anonfun$apply$4.apply(routes_routing.scala:85) ~[na:na]
	at play.core.Router$HandlerInvoker$$anon$7$$anon$2.invocation(Router.scala:183) ~[play_2.10.jar:2.2.3]
	at play.core.Router$Routes$$anon$1.invocation(Router.scala:377) ~[play_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$1.call(JavaAction.scala:56) ~[play_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction$1.apply(TransactionalAction.java:20) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction$1.apply(TransactionalAction.java:18) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.JPA.withTransactionAsync(JPA.java:177) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.db.jpa.TransactionalAction.call(TransactionalAction.java:15) ~[play-java-jpa_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:91) ~[play_2.10.jar:2.2.3]
	at play.core.j.JavaAction$$anon$3.apply(JavaAction.scala:90) ~[play_2.10.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.3]
	at play.core.j.FPromiseHelper$$anonfun$flatMap$1.apply(FPromiseHelper.scala:82) ~[play_2.10.jar:2.2.3]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251) ~[scala-library.jar:na]
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:249) ~[scala-library.jar:na]
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:32) [scala-library.jar:na]
	at play.core.j.HttpExecutionContext$$anon$2.run(HttpExecutionContext.scala:37) ~[play_2.10.jar:2.2.3]
	at akka.dispatch.TaskInvocation.run(AbstractDispatcher.scala:42) ~[akka-actor_2.10.jar:2.2.0]
	at akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask.exec(AbstractDispatcher.scala:386) ~[akka-actor_2.10.jar:2.2.0]
	... 4 common frames omitted
Caused by: org.hibernate.TransientObjectException: object references an unsaved transient instance - save the transient instance before flushing: models.User
	at org.hibernate.engine.internal.ForeignKeys.getEntityIdentifierIfNotUnsaved(ForeignKeys.java:249) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.type.EntityType.getIdentifier(EntityType.java:510) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.type.ManyToOneType.nullSafeSet(ManyToOneType.java:165) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.persister.collection.AbstractCollectionPersister.writeElement(AbstractCollectionPersister.java:863) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.persister.collection.AbstractCollectionPersister.insertRows(AbstractCollectionPersister.java:1471) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.persister.collection.OneToManyPersister.insertRows(OneToManyPersister.java:190) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.action.internal.CollectionUpdateAction.execute(CollectionUpdateAction.java:86) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.engine.spi.ActionQueue.execute(ActionQueue.java:377) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:369) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.engine.spi.ActionQueue.executeActions(ActionQueue.java:291) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.event.internal.AbstractFlushingEventListener.performExecutions(AbstractFlushingEventListener.java:339) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.event.internal.DefaultFlushEventListener.onFlush(DefaultFlushEventListener.java:52) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.internal.SessionImpl.flush(SessionImpl.java:1234) ~[hibernate-core-4.2.3.Final.jar:4.2.3.Final]
	at org.hibernate.ejb.AbstractEntityManagerImpl.flush(AbstractEntityManagerImpl.java:996) ~[hibernate-entitymanager-4.2.3.Final.jar:4.2.3.Final]
	... 25 common frames omitted

